_data/people/sgnoohc.yml

```
@@ -0,0 +1,53 @@
+active: true
+focus-area:
+- ia
+institution: University of California, San Diego
+name: Philip Chang
+photo: "/assets/images/team/Philip-Chang.jpg"
+shortname: sgnoohc
+title: Tracking Algorithms Collaborator
+website: philip.ucsd.edu
```

```suggestion
website: https://philip.ucsd.edu
```

Just needs the full URL